src/session.ts

```diff
@@ -222,18 +239,25 @@ class Session extends ServiceObject {
    * @example
    * session.beginTransaction(function(err, transaction, apiResponse) {});
    */
-  beginTransaction(options, callback) {
-    if (is.fn(options)) {
-      callback = options;
-      options = {};
-    }
+  // tslint:disable-next-line no-any
+  beginTransaction(options: any): Promise<BeginTransactionResponse>;
```

Comment:
Let's not use any here, and define a type instead :)

Reply:
Thank you @JustinBeckwith . I did the change.